internal/lightcone/hunt/onlysilenceremains/onlysilenceremains.go

```diff
@@ -0,0 +1,72 @@
+package onlysilenceremains
+
+import (
+	"github.com/simimpact/srsim/pkg/engine"
+	"github.com/simimpact/srsim/pkg/engine/equip/lightcone"
+	"github.com/simimpact/srsim/pkg/engine/event"
+	"github.com/simimpact/srsim/pkg/engine/info"
+	"github.com/simimpact/srsim/pkg/engine/modifier"
+	"github.com/simimpact/srsim/pkg/engine/prop"
+	"github.com/simimpact/srsim/pkg/key"
+	"github.com/simimpact/srsim/pkg/model"
+)
+
+const (
+	OnlySilenceRemains       key.Modifier = "only_silence_remains"
+	OnlySilenceRemainsCRBuff key.Modifier = "only_silence_remians_cr_buff"
+)
+
+// Increases ATK of its wearer by 16/20/24/28/32%. If there are 2 or fewer
+// enemies on the field, increases wearer's CRIT Rate by 12/15/18/21/24%.
+func init() {
+	lightcone.Register(key.OnlySilenceRemains, lightcone.Config{
+		CreatePassive: Create,
+		Rarity:        4,
+		Path:          model.Path_HUNT,
+		Promotions:    promotions,
+	})
+
+	modifier.Register(OnlySilenceRemains, modifier.Config{
+		Listeners: modifier.Listeners{
+			OnTriggerDeath: onTriggerDeath,
+		},
+	})
+
+	modifier.Register(OnlySilenceRemainsCRBuff, modifier.Config{
+		Stacking:   modifier.ReplaceBySource,
+		StatusType: model.StatusType_STATUS_BUFF,
+	})
```

Comment:
1. for whatever reason, this modifier is not registered as a `BUFF` in game (so dont want to label it as a buff here)
2. I would leave the `Stacking` unspecified (default is `Unique`). The reason for this is repeat applications when using `ReplaceBySource` will remove the old modifier and add the new version. This will create a lot of noise in the logs, when in reality the modifier add calls will always be identical. Keeping it as `Unique` means `AddModifier` will only add the modifier if it is not already present.